pydantic/json_schema.py

```diff
@@ -1877,8 +1877,8 @@ def complex_schema(self, schema: core_schema.ComplexSchema) -> JsonSchemaValue:
 
         JSON has no standard way to represent complex numbers. Complex number is not a numeric
         type. Here we represent complex number as strings following the rule defined by Python.
-        For instance, '1+2j' is an accepted complex string. Details can be found at
-        https://docs.python.org/3/library/functions.html#complex
+        For instance, '1+2j' is an accepted complex string. Details can be found in
+        [Python's `complex` documentation](https://docs.python.org/3/library/functions.html#complex).
```

Comment:
```suggestion
        For instance, '1+2j' is an accepted complex string. Details can be found in
        [Python's `complex` documentation][complex].
```
I'd rather use crossrefs. Although it does look a bit unclear where it comes from. Sphinx [allows explicit targets](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#role-external) but mkdocstrings is still quite limited here :face_with_diagonal_mouth: